tests/BlockChypTest/Client/CryptoTest.cs

```diff
@@ -44,22 +43,13 @@ public void CryptoTest_GenerateAuthHeaders()
             Assert.NotNull(result["Authorization"]);
         }
 
-        [Fact]
-        public void CryptoTest_EncryptDecrypt()
+        /// <summary>
+        /// Test that the <c>Crypto.Encrypt</c> and <c>Crypto.Decrypt</c> methods round-trip.
+        /// </summary>
+        [Property(MaxTest = 10000)]
+        public bool prop_EncryptDecrypt_RoundTrip(NonNull<string> plaintext, AesKey aesKey)
         {
-            byte[] key;
-            using (Aes aes = Aes.Create())
-            {
-                key = aes.Key;
-            }
-
-            var plaintext = "some super secret sauce";
-
-            var encrypted = Crypto.Encrypt(plaintext, key);
-
-            var roundTrip = Crypto.Decrypt(encrypted, key);
-
-            Assert.Equal(plaintext, roundTrip);
+            return plaintext.Get == Crypto.Decrypt(Crypto.Encrypt(plaintext.Get, aesKey.Get), aesKey.Get);
```

Comment:
I'd prefer to see encrypt, decrypt, and the assertion on separate lines. It makes it more clear what is being tested.

Reply:
Done.